project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -3608,3 +3615,366 @@ function c4m_og_entity_delete($entity, $entity_type) {
     taxonomy_vocabulary_delete($tags->vid);
   }
 }
+
+/**
+ * Returns content type labels.
+ *
+ * @param array $machine_names
+ *   Content type machine names.
+ *
+ * @return array
+ *   Returns array of content type labels.
+ */
+function c4m_og_get_content_type_labels(array $machine_names) {
+  $query = db_select('node_type', 'nt')
+    ->fields('nt', array('type', 'name'))
+    ->condition('type', $machine_names, 'IN');
+  $results = $query->execute()->fetchAll();
+  $content_types = [];
+  foreach ($results as $result) {
+    $content_types[$result->type] = $result->name;
+  }
+  return $content_types;
+}
+
+/**
+ * Form builder.
+ *
+ * Merge group content.
+ */
+function c4m_og_merge_groups_build_form($form, $form_state, $group) {
+  $current_group = c4m_og_current_group();
+  $content_types = c4m_og_feature_enabled_content_types();
+  if (!empty($content_types)) {
+    $options = c4m_og_get_content_type_labels($content_types);
+    $form['content_types'] = array(
+      '#title' => t('Select the different content types to merge to the target group. Note that all other  (unselected) content will be deleted permanently.'),
+      '#type' => 'checkboxes',
+      '#options' => $options,
+      '#default_value' => $content_types,
+    );
+  }
+
+  $form['users'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Merge users'),
+    '#default_value' => 1,
+  );
+
+  $form['target_group'] = array(
+    '#type' => 'textfield',
+    '#title' => t("Merge to"),
+    '#entityreference' => array(
+      'target_type' => 'node',
+      'handler' => 'c4m_selection',
+      'type' => 'single',
+      'handler_settings' => array(
+        'target_bundles' => array('group' => $current_group->type),
+        'sort' => array(
+          'direction' => 'ASC',
+          'property' => 'title',
+          'type' => 'property',
+        ),
+      ),
+    ),
+  );
+
+  $form['#validate'][] = '_c4m_og_merge_groups_validation';
+
+  $form['merge'] = array(
+    '#type' => 'submit',
+    '#value' => 'Merge',
+  );
+
+  return $form;
+}
+
+/**
+ * Validates group merge form.
+ */
+function _c4m_og_merge_groups_validation($form, &$form_state) {
+  $current_group = og_context();
```

Comment:
This is not available. Can't we filter out the current group on autocomplete?

Reply:
This can't be done from params or hook alters.